File: app/src/main/java/com/example/floatingactionbutton/MiniFloatingActionButton.java
package com.example.floatingactionbutton;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

import com.google.android.material.floatingactionbutton.FloatingActionButton;

public class MiniFloatingActionButton extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_mini_floating_action_button);

        FloatingActionButton miniFab = findViewById(R.id.mini_fab_btn);

        miniFab.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MiniFloatingActionButton.this, "Clicked on Mini FAB Button", Toast.LENGTH_SHORT).show();
            }
        });
    }
}

File: app/src/main/java/com/example/floatingactionbutton/MiniFloatingActionButton.kt
package com.example.floatingactionbutton

import android.os.Bundle
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity
import com.google.android.material.floatingactionbutton.FloatingActionButton

class MiniFloatingActionButton : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_mini_floating_action_button)
        val miniFab = findViewById<FloatingActionButton>(R.id.mini_fab_btn)
        miniFab.setOnClickListener { Toast.makeText(this@MiniFloatingActionButton, "Clicked on Mini FAB Button", Toast.LENGTH_SHORT).show() }
    }
}